docs/mdbook/src/android/geojson-guide.md

```diff
@@ -0,0 +1,373 @@
+# Introduction
+
+This guide will teach you how to use [`GeoJsonDataSource`]("https://maplibre.org/maplibre-native/android/api/-map-libre%20-native%20-android/org.maplibre.android.style.sources/-geo-json-source/index.html") by deep diving into [`GeoJson`]("https://geojson.org/") file format.
```

Comment:
Names that use own names such as GeoJSON (note capitalization) and JSON should not use backticks. That should only be done for names of classes / variables.

Reply:
This conversation was resolved, but there are still instances where GeoJSON and other names of things that are not classes or variable names use backticks.